src/App.tsx

```diff
@@ -15,36 +16,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  LENGTH = 'length',
+  ALPHABET = 'alphabet',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: string,
+  isReversed: boolean,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    switch (sortField) {
+      case SortType.ALPHABET:
+        preparedGoods.sort((good1, good2) => good1.localeCompare(good2));
+        break;
+      case SortType.LENGTH:
+        preparedGoods.sort((good1, good2) => good1.length - good2.length);
+        break;
+      default:
+        return [];
+    }
+  }
+
+  if (isReversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [isReversed, setReversed] = useState(false);
+  const [sortField, setSortField] = useState('');
```

Comment:
`''` - add this as default value in enum
```suggestion
  const [sortField, setSortField] = useState(SortType.DEFAULT);
```